assets/js/app.js

```diff
@@ -87,12 +87,11 @@ window.addEventListener('phx:page-loading-stop', () => {
 
 window.addEventListener('keydown', event => {
   const currentURL = window.location.pathname;
-  const edit_job_url = /\/projects\/(.+)\/w\/(.+)\/j\/(.+)/;
+  const edit_workflow_url = /\/projects\/(.+)\/w\/(.+)/;
   if ((event.ctrlKey || event.metaKey) && event.key === 's') {
-    if (edit_job_url.test(currentURL)) {
+    if (edit_workflow_url.test(currentURL)) {
       event.preventDefault();
-      console.log('Saving the job');
-      let form = document.querySelector("button[form='job-form']");
+      let form = document.querySelector("button[form='workflow-form']");
       form.click();
     }
   }
```

Comment:
done @stuartc ! (I hope I'm not missing something in the implementation, but following the fly.io example this was actually a super fast change.)

please note that when we use this hook approach (which I agree is 100% the right way to go!) rather than the listener-plus-route approach this behaviour breaks when someone clicks an edge. CC @josephjclark , I think this is good because we'll sort out the broken edge clicks in another branch.

so... in so far as the form works as expected, this new hook works as expected for `ctrl-s` and `command-s`.